webapps/ui/cockpit/client/scripts/camunda-cockpit-ui.js

```diff
@@ -77,15 +77,20 @@ module.exports = function(pluginDependencies) {
     '$locationProvider',
     '$animateProvider',
     '$qProvider',
+    '$compileProvider',
     function(
       $routeProvider,
       UriProvider,
       $modalProvider,
       $tooltipProvider,
       $locationProvider,
       $animateProvider,
-      $qProvider
+      $qProvider,
+      $compileProvider
     ) {
+      $compileProvider.aHrefSanitizationWhitelist(
```

Comment:
According to [docs](https://docs.angularjs.org/api/ng/provider/$compileProvider#aHrefSanitizationWhitelist) this method is deprecated, it says use `aHrefSanitizationTrustedUrlList` instead.

Reply:
Also, I doubt that this method is ever removed since AngularJS is out of maintenance, I agree that it is not a good idea to use deprecated methods. Wasn't aware of it. Nice catch! 🙂 👍